src/components/Header/components/LocationPopUp/LocationPopUp.jsx

```diff
@@ -0,0 +1,101 @@
+import Select, { components } from "react-select";
+import Button from "../NavigationBar/Button/Button";
+import "./LocationPopUp.css";
+
+const controlStyles = {
+  border: "1px solid rgb(226, 223, 223)",
+  padding: "25px 10px 10px 10px",
+  background: "white",
+  color: "white",
+  borderRadius: "10px 10px 0 0",
+};
+
+export default function LocationPopUp({
+  options,
+  selectedCities,
+  onChange,
+  onSave,
+}) {
+  const multiValueContainer = ({ selectProps, data }) => {
```

Comment:
It is better to name React components in Capitalized style
```suggestion
  const MultiValueContainer = ({ selectProps, data }) => {
```